ethereum/ethereum_type.bal

```diff
@@ -0,0 +1,351 @@
+    # + position - The uncle's index position
+    # + return - If success, return string, else returns Error
+    public function getEthUncleByBlockNumberAndIndex(string block, string position) returns string|Error;
+
+    # Creates a filter object, based on filter options, to notify when the state changes (logs).
+    #
+    # + filterOptions - The filter options
+    # + return - If success, return string, else returns Error
+    public function ethNewFilter(json filterOptions) returns string|Error;
+
+    # Creates a filter in the node, to notify when a new block arrives.
+    #
+    # + return - If success, return string, else returns Error
+    public function ethNewBlockFilter() returns string|Error;
+
+    # Creates a filter in the node, to notify when new pending transactions arrive.
+    #
+    # + return - If success, return string, else returns Error
+    public function ethNewPendingTransactionFilter() returns string|Error;
+
+    # Uninstalls a filter with given id.
+    #
+    # + filterId - The filter id
+    # + return - If success, return string, else returns Error
+    public function uninstallEthFilter(string filterId) returns string|Error;
+
+    # Get an array of logs which occurred since last poll.
+    #
+    # + filterId - The filter id
+    # + return - If success, return string, else returns Error
+    public function getEthFilterChanges(string filterId) returns string|Error;
+
+    # Get an array of all logs matching filter with given id.
+    #
+    # + filterId - The filter id
+    # + return - If success, return string, else returns Error
+    public function getEthFilterLogs(string filterId) returns string|Error;
+
+    # Get an array of all logs matching a given filter object.
+    #
+    # + filterOptions - The filter options
+    # + return - If success, return string, else returns Error
+    public function getEthLogs(json filterOptions) returns string|Error;
+
+    # Get the hash of the current block, the seedHash, and the boundary condition to be met ("target").
+    #
+    # + return - If success, return string, else returns Error
+    public function getEthWork() returns string|Error;
+
+    # Used for submitting a proof-of-work solution.
+    #
+    # + nonce - The nonce found (64 bits)
+    # + powHash - The header's pow-hash (256 bits)
+    # + mixDigest - The mix digest (256 bits)
+    # + return - If success, return string, else returns Error
+    public function submitEthWork(string nonce, string powHash, string mixDigest) returns string|Error;
+
+    # Generic method for Ethereum RPC.
+    #
+    # + method - The method name
+    # + params - The parameter array
+    # + return - If success, return string, else returns Error.
+    public function remoteProcedureCall(string method, json params) returns string|Error;
+};
+
+# Ethereum error.
+#
+# + message - Error message.
+# + cause - The error which caused the Ethereum error.
+# + statusCode - The status code.
+public type Error record {
```

Comment:
What I meant was not to rename this record, But to use ballerina builtin `error` type instead of this one.See [1] for example.
[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/stdlib/http/src/main/ballerina/http/http_client.bal#L36